src/blueprints/tv2/tv2-blueprint-configuration-validator.ts

```diff
@@ -39,4 +40,50 @@ export class Tv2BlueprintConfigurationValidator implements BlueprintValidateConf
         })
     })
   }
-}
+
+  private validateGraphicsDefault(showStyleConfiguration: Tv2ShowStyleBlueprintConfiguration): StatusMessage[] {
+    const validationErrors: StatusMessage[] = []
+  
+    // Check if showStyleConfiguration.graphicsDefault exists
+    if (!showStyleConfiguration.graphicsDefault) {
+      validationErrors.push({
+        id: 'GraphicsDefaultMissing',
+        title: 'Graphics Default Configuration',
+        message: 'Graphics default configuration is missing',
+        statusCode: StatusCode.BAD
+      })
+    } else {
+      // Check if DefaultSetupName is empty
+      if (!showStyleConfiguration.graphicsDefault.setupName?.value) {
```

Comment:
We want to reduce nesting as much as possible. Instead of doing the nested if-statements you could return early from your first if-statement:
```
private validateGraphicsDefault(...) {
  if (!showStyleConfiguration.graphicsDefault) {
    return [{
      id: '',
      ...
    }]
  }

  const validationErrors: StatusMessage[] = []
  // Rest of your if-statements from here.
}
```

We useally refer to returning early as "defensive programming" (I am aware that that term covers more than that)